### ISSUE TITLE
Support uploading bigger files with createWriteStream

### ISSUE THREAD

**Issue by Paxa** [perry-mitchell/webdav-client#144]
axios uses follow-redirect, which limit body size to 10 MB (for some unknown reasons)
So webdav-client uploading is limited to 10 MB also, and there is no way to specify custom limit:

This example doesn't work because only `httpAgent`, `httpsAgent`, `headers` and `withCredentials` allowed
```js
client.createWriteStream(davPath, {maxContentLength: 1024 ** 3}); // 1 GB
// Error: Request body larger than maxBodyLength limit
```

I think webdav-client should allow any size for uploading files or make it configurable, may be something like global axios settings

**Comment by perry-mitchell** [comment]
Yeah this seems to be an axios flaw like you said. Would be good to add it as an option in this library, with it being auto-set at say 1GB by default. As you showed setting `maxContentLength` in axios options will do the trick.

**Comment by sean256** [comment]
@perry-mitchell I submitted a PR for exactly this

**Comment by perry-mitchell** [comment]
This should be fixed by #146

**Comment by perry-mitchell** [comment]
Released in 2.6.0
